addons/mass_mailing/models/mailing_trace.py

```diff
@@ -98,7 +98,9 @@ class MailingTrace(models.Model):
         ("mail_bl", "Blacklisted Address"),
         ("mail_optout", "Opted Out"),
         ("mail_dup", "Duplicated Email"),
+        ("mail_bounce", "Email bounced"),
```

Comment:
Would order alphabetically in the "mass mode" section (time to move mail_dup while we are at it).